### PR TITLE
operators [N] [CI] synapse-helm (0.0.1)

### PR DESCRIPTION
**Pull request by mgoerens**
Signed-off-by: mgoerens <mgoerens@redhat.com>

Thanks submitting your Operator. Please check below list before you create your Pull Request.

### New Submissions

* [x] Are you familiar with our [contribution guidelines](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-via-pr.md)?
* [x] Have you [packaged and deployed](https://github.com/operator-framework/community-operators/blob/master/docs/testing-operators.md) your Operator for Operator Framework?
* [x] Have you tested your Operator with all Custom Resource Definitions?
* [x] Have you tested your Operator in all supported [installation modes](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/design/building-your-csv.md#operator-metadata)?
* [x] Have you considered whether you want use [semantic versioning order](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md#semver-mode)?
* [x] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?
* [x] Is operator [icon](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#operator-icon) set?

### Updates to existing Operators

* [ ] Did you create a `ci.yaml` file according to the [update instructions](https://github.com/operator-framework/community-operators/blob/master/docs/operator-ci-yaml.md)?
* [ ] Is your new CSV pointing to the previous version with the `replaces` property if you chose `replaces-mode` via the `updateGraph` property in `ci.yaml`?
* [ ] Is your new CSV referenced in the [appropriate channel](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#channels) defined in the `package.yaml` or `annotations.yaml` ?
* [ ] Have you tested an update to your Operator when deployed via OLM?
* [ ] Is your submission [signed](https://github.com/operator-framework/community-operators/blob/master/docs/contributing-prerequisites.md#sign-your-work)?

### Your submission should not

* [ ] Modify more than one operator
* [ ] Modify an Operator you don't own
* [ ] Rename an operator - please remove and add with a different name instead
* [ ] Modify any files outside the above mentioned folders
* [ ] Contain more than one commit. **Please squash your commits.**

### Operator Description must contain (in order)

1. [ ] Description about the managed Application and where to find more information
2. [ ] Features and capabilities of your Operator and how to use it
3. [ ] Any manual steps about potential pre-requisites for using your Operator

### Operator Metadata should contain

* [ ] Human readable name and 1-liner description about your Operator
* [ ] Valid [category name](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-operator.md#categories)<sup>1</sup>
* [ ] One of the pre-defined [capability levels](https://github.com/operator-framework/operator-courier/blob/4d1a25d2c8d52f7de6297ec18d8afd6521236aa2/operatorcourier/validate.py#L556)<sup>2</sup>
* [ ] Links to the maintainer, source code and documentation
* [ ] Example templates for all Custom Resource Definitions intended to be used
* [ ] A quadratic logo

Remember that you can preview your CSV [here](https://operatorhub.io/preview).

--

<sup>1</sup> If you feel your Operator does not fit any of the pre-defined categories, file an issue against this repo and explain your need

<sup>2</sup> For more information see [here](https://sdk.operatorframework.io/docs/overview/#operator-capability-level)

### CODE REVIEW

**Comment by mgoerens** [comment]
First contribution, and first time creating and packaging an operator. Therefore.. comments are more than welcomed !

File: operators/synapse-helm/0.0.1/scorecard/config.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+apiVersion: scorecard.operatorframework.io/v1alpha3
```
Comment (mvalarh) [inline operators/synapse-helm/0.0.1/scorecard/config.yaml:1]: @mgoerens please move file `operators/synapse-helm/0.0.1/scorecard/config.yaml` to `operators/synapse-helm/0.0.1/tests/scorecard/config.yaml`

**Review by mvalarh — CHANGES_REQUESTED** [review]

File: operators/synapse-helm/0.0.1/scorecard/config.yaml (hunk 1 above)
Comment (mgoerens) [reply operators/synapse-helm/0.0.1/scorecard/config.yaml]: Thanks Martin, done!

**Review by mvalarh — APPROVED** [review]

**Comment by J0zi** [comment]
@mgoerens Thaank you for contriguting. As you see, operator installation have failed https://github.com/k8s-operatorhub/community-operators/pull/23/checks?check_run_id=3170424432#step:3:2093
Please also add couple sentences to description, you can check how it displays in the [preview](https://operatorhub.io/preview).

**Comment by mgoerens** [comment]
> @mgoerens Thaank you for contriguting. As you see, operator installation have failed https://github.com/k8s-operatorhub/community-operators/pull/23/checks?check_run_id=3170424432#step:3:2093

I just pushed some changes which hopefully will fix the issue. I wasn't able to run those tests locally, which explains the multiple errors.

> Please also add couple sentences to description, you can check how it displays in the [preview](https://operatorhub.io/preview).

I will do so next !

**Comment by mgoerens** [comment]
I wrote a short description, and implemented additional changes to try to get the pipeline green.

**Comment by mgoerens** [comment]
@frzifus https://github.com/opdev/synapse-helm

**Pull request merged**